OpenChain Telco SBOM Specification.md

```diff
@@ -12,7 +12,7 @@ This specification is licensed under [Creative Commons Attribution License 4.0 (
 
 ## 2. Terms and definitions
 
-### Data Format
+### 
```

Comment:
Did you mean to remove the title here?

Reply:
I did not, I blame staying up past my bedtime doing work ;)